libs/libuv/stone.yml

```diff
@@ -0,0 +1,26 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : libuv
```

Comment:
Looks like there's a man page for this package.

Reply:
We need python-sphinx for docs, added a TODO note.